platform-specific/dtls_prng_posix.c

```diff
@@ -37,7 +37,31 @@ int
 dtls_prng(unsigned char *buf, size_t len) {
 #ifdef HAVE_GETRANDOM
   return getrandom(buf, len, 0);
-#else /* !HAVE_GETRANDOM */
+#elif defined(HAVE_RANDOM)
+
+#define RAND_BYTES (RAND_MAX > 0xffffff ? 2 : (RAND_MAX > 0xffff ? 1 : 0))
+
+  if (len) {
+    size_t klen = len;
+    uint8_t byte_counter = RAND_BYTES;
+    uint32_t rand = random();
+    while (1) {
+      *buf++ = rand & 0xFF;
+      if (!--klen) {
+        break;
+      }
+      if (byte_counter--) {
```

Comment:
I think the following is better and more readable (as RAND_BYTES is what it says on the tin)
````
#define RAND_BYTES (RAND_MAX >= 0xffffff ? 3 : (RAND_MAX >= 0xffff ? 2 : 1))

....

      if (--byte_counter) {
````

Reply:
Thanks. Done.